conda_recipe_manager/commands/bump_recipe.py

```diff
@@ -15,11 +15,48 @@
 from conda_recipe_manager.parser.recipe_parser import RecipeParser
 from conda_recipe_manager.types import JsonPatchType
 
-
 # TODO Improve. In order for `click` to play nice with `pyfakefs`, we set `path_type=str` and delay converting to a
 # `Path` instance. This is caused by how `click` uses decorators. See these links for more detail:
 # - https://pytest-pyfakefs.readthedocs.io/en/latest/troubleshooting.html#pathlib-path-objects-created-outside-of-tests
 # - https://github.com/pytest-dev/pyfakefs/discussions/605
+
+
+def _get_required_patch_blob(recipe_parser: RecipeParser, increment_build_num: bool) -> JsonPatchType:
+    """
+    Returns the required JSON Patch Blob
+
+    :recipe_parser: RecipeParser object
+    :build_num: `build_num` boolean flag
```

Comment:
```suggestion
    :param recipe_parser: Recipe file to update.
    :param increment_build_num: Increments the `/build/number` field by 1 if set to `True`. Otherwise resets to 0.
```